### PR TITLE
Testcases part3

### PR DESCRIPTION
**Pull request by jkingoliver**
Watson Conversation
Redis
Postgre

### CODE REVIEW

File: test/app/postgre/node/server.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            return;
+        }
+
+        client.query('INSERT INTO "sch.test" (var1, var2) VALUES ($1, $2)', [pair[0], pair[1]], function (err, result) {
+            if (err) {
+                res.status(500).send(err);
+                return;
+            }
+        });
+        client.query('SELECT * FROM "sch.test";', function (err, result) {
```
Comment (gabe-terrell) [inline test/app/postgre/node/server.js]: My only concern here is that this doesn't actually validate if the data we just inserted is the same data we're getting back.

File: test/app/redis/node/server.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	if (!redis) {
+		res.status(500).send('redis is not defined in serviceManager');
+		return;
+	}
+
+	redis.set("test-key", "test-val");
+
+	messages.push("set data");
+
+	redis.get("test-key", function (err, response) {
```
Comment (gabe-terrell) [inline test/app/redis/node/server.js]: Same here. This doesn't actually validate if the data we just inserted is the same data we're getting back.

Comment (jkingoliver) [reply test/app/redis/node/server.js]: I followed the pattern that most of the other node and python tests follow. If we want more extensive tests then we'll need to address all services.

**Comment by jkingoliver** [comment]
@gabe-terrell Please see updates to address your comments.

**Pull request merged**